docs/plugins.asciidoc

```diff
@@ -13,24 +13,18 @@ A list of existing Kibana plugins is available on https://github.com/elastic/kib
 Use the following command to install a plugin:
 
 [source,shell]
-bin/kibana plugin --install <org>/<package>/<version>
+bin/kibana-plugin install <package> (optional URL)
 
-You can also use `-i` instead of `--install`, as in the following example:
-
-[source,shell]
-bin/kibana plugin -i elasticsearch/marvel/latest
-
-Because the organization given is `elasticsearch`, the plugin management tool automatically downloads the 
-plugin from `download.elastic.co`.
+When you use `elasticsearch` as the organization name, the plugin management tool downloads the plugin from 
```

Comment:
There is no concept of organization name any longer.  Plugins are either installed by url or by name.  Any time you try to install by name, it will download from `download.elastic.co`.